shell/platform/darwin/ios/framework/Source/FlutterDartVMServicePublisher.mm

```diff
@@ -83,18 +83,18 @@ - (void)publishServiceProtocolPort:(NSURL*)url {
   // Physical devices need to request all interfaces.
   uint32_t interfaceIndex = 0;
 #endif  // TARGET_IPHONE_SIMULATOR
-  const char* registrationType = "_dartobservatory._tcp";
+  const char* registrationType = "_dartVmService._tcp";
```

Comment:
@jmagman I'd like to change the mDNS entry name to stop referencing Observatory. Are there any gotchas I should be aware of? Should we be supporting both `_dartVmService._tcp` and `_dartobservatory._tcp` until flutter_tools is updated?

Reply:
Yes, you'll need to support both until the tool is updated or the engine->flutter roll will have a wall of test failures.  I don't see any usages in g3 and I can't think of gotchas outside of the tool (please add me as a reviewer when you get to that PR).

cc @vashworth who is working on that area of the tool at the moment https://github.com/flutter/flutter/pull/118104

Reply:
Makes sense. I have a WIP PR ready for updating the tool here: https://github.com/flutter/flutter/pull/118577, so I'll add you to that PR once this change lands and I've confirmed things are working as expected.